tests/envs/test_playroom.py

```diff
@@ -213,6 +213,13 @@ def test_playroom_simulate_doors_and_dial():
     assert np.any(state[robot] != next_state[robot])
     assert np.any(state[door1] != next_state[door1])
     state = next_state
+    # Move to playroom door
+    act = Action(np.array([110.2, 15, 3, 1, 1]).astype(np.float32))
+    next_state = env.simulate(state, act)
+    for o in state:
+        if o.type != robot_type:
+            assert np.all(state[o] == next_state[o])
```

Comment:
Might want to use allclose here for safety against precision issues? `np.allclose(state[o], next_state[o])`

Reply:
Do you want to also add an `else: assert not np.allclose(state[o], next_state[o])`?

Reply:
good idea, thanks